server/src/routers/userRouter.js

```diff
@@ -1,21 +1,31 @@
-const express = require('express')
-const path = require('path')
+const express = require('express');
 const router = new express.Router()
 
-const User = require('../models/users')
+const User = require("../models/users")
 
-router.post('/signup', async (req, res) => {
-    console.log('Creating a new account for a new user')
+router.post("/user/signup", async (req, res) => {
```

Comment:
According to the REST API convention, I think we should use the route as `/users/signup` instead of `/user/signup`

Reply:
I kept it to user/signup for two reasons.
1. At a point, only one user is signing up so this endpoint is taking care of signing up process of one user.
2. To keep this endpoint in sync with all other user's endpoints where it is user/*